test/extended/networking/egressip_helpers.go

```diff
@@ -1930,28 +1930,31 @@ func sdnHostsubnetFlushEgressCIDRs(oc *exutil.CLI, nodeName string) error {
 // runOcWithRetry runs the oc command with up to 5 retries if a timeout error occurred while running the command.
 func runOcWithRetry(oc *exutil.CLI, cmd string, args ...string) (string, error) {
 	var err error
-	var output string
+	var stdout string
 	maxRetries := 5
 
 	for numRetries := 0; numRetries < maxRetries; numRetries++ {
 		if numRetries > 0 {
 			framework.Logf("Retrying oc command (retry count=%v/%v)", numRetries+1, maxRetries)
 		}
 
-		output, err = oc.Run(cmd).Args(args...).Output()
+		// stderrr can have spurious logs that can disrupt parsing done by
+		// callers, ignore it
+		stdout, _, err = oc.Run(cmd).Args(args...).Outputs()
 		// If an error was found, either return the error, or retry if a timeout error was found.
 		if err != nil {
 			if strings.Contains(strings.ToLower(err.Error()), "i/o timeout") {
 				// Retry on "i/o timeout" errors
 				framework.Logf("Warning: oc command encountered i/o timeout.\nerr=%v\n)", err)
 				continue
 			}
-			return output, err
+			return stdout, err
```

Comment:
Shall we return when an error occurs? How about log it, then retry?

Reply:
I don't mean we shouldn't ignore stderr.
The function is named `runOcWithRetry`. It implies that `retry until succeed` to me. So I prefer that we ignore all types of errors until it succeeds or reaches the `maxRetries`.

Reply:
That's what its doing. It is retrying until it succeeds (only for timeout errors, as documented). I am not changing that behavior in any way.  I don't understand what you mean.